Dockerfile

```diff
@@ -10,13 +10,13 @@ COPY consoles/                              /usr/share/prometheus/consoles/
 RUN ln -s /usr/share/prometheus/console_libraries /usr/share/prometheus/consoles/ /etc/prometheus/
 RUN mkdir -p /prometheus && \
     chown -R nobody:nogroup etc/prometheus /prometheus
+RUN ln -s /prometheus /data
+RUN ln -s /etc/prometheus/prometheus.yml /prometheus.yml
+RUN ln -s /etc/prometheus/console_libraries /console_libraries
```

Comment:
`/usr/share/prometheus/console_libraries` rather?

Reply:
I thought we were intending to move those to `/etc/prometheus/...`

Reply:
Sorry I'm confused

Reply:
We already setup a symlinks from 1/usr/share/prometheus/...` to `/etc/prometheus/...` above on line 10.